cloud-sql/mysql/mysql/connect-unix.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+// [START cloud_sql_mysql_mysql_connect_unix]
+const mysql = require('promise-mysql');
+
+const createUnixSocketPool = async config => {
+  const dbSocketPath = process.env.INSTANCE_UNIX_SOCKET
```

Comment:
I think we should require users to set a `INSTANCE_UNIX_SOCKET` and remove this ternary expression. See below about requiring one or the other env var.

Reply:
Done.